src/components/PersonLink.tsx

```diff
@@ -0,0 +1,56 @@
+import { Link, useParams } from 'react-router-dom';
+import cn from 'classnames';
+import { Person } from '../types';
+
+interface Props {
+  person: Person;
+  names: string[];
+  findParent: (el: string) => Person | undefined;
+}
+
+export const PersonLink: React.FC<Props> = ({ person, names, findParent }) => {
```

Comment:
the name of the component does not completly match its content

its more like `TableContent` or `TableData`